client/app/dbaas/logs/detail/home/capped/logs-home-capped.html

```diff
@@ -0,0 +1,40 @@
+<form novalidate name="ctrl.form">
+    <oui-modal data-title="{{ ::'logs_update_capped_title' | translate }}"
+           data-primary-action="ctrl.updateCappedPlan()"
+           data-primary-label="{{ ::'common_save' | translate }}"
+           data-secondary-action="ctrl.cancel()"
+           data-secondary-label="{{ ::'common_cancel' | translate }}"
+           data-on-dismiss="ctrl.cancel()"
+           data-loading="ctrl.saving.loading"
+           data-ng-if="ctrl.offer != 'logs-basic'">
+        <div class="oui-field">
+            <div class="oui-field__content">
+                <div class="oui-field-control oui-field-control_5">
+                    <oui-checkbox
+                        id="service_is_capped"
```

Comment:
No! Prefer `service-is-capped` for instance.

Since we use the BEM naming convention on the OVH UI Kit, it make more sense to stay with lowercase and hyphens.